### PR TITLE
ELB Refactor for ssl_certificate_id, elb_health_check, ports and protocols  AUTO-739

### PR DESCRIPTION
**Pull request by fuddstar**
Big Changes to elbs/elb_config

* elb_health_check (explicit path) will replace path2ping (constructed path) **breaking change**
path2ping: /index.html
elb_health_check: HTTP:80/index/html
* Introducing instance_protocol and loadbalancer_protocol replacing protocols **breaking change**
* Replacing instanceports with instance_port **breaking change**
* Replacing loadbalancerports with loadbalancer_port **breaking change**
* All unit and systems tests updated

Passing Build: https://gaautobots.atlassian.net/builds/browse/AM-AUT59/latest

### CODE REVIEW

**Comment by GypsyBojangles** [comment]
Approved

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
...
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.3.0] - 24/08/2016
+- Introducing SSLCertificateId for ELBs
+- Changing path2ping to elb_health_check and making it an explicit health check e.g. /index.html --> HTTP:80/index.html. Updates to app and default yamls required 
+- Changing instance_protocol and loadbalancer_protocol to replace protocols. Updates to app and default yamls required 
+- Changing loadbalancerports to loadbalancer_port. Updates to app and default yamls required 
+- Changing instanceports to instance_port. Updates to app and default yamls required 
+- Updated relavant unit tests
+- Updated relavant systems tests
```
Comment (GypsyBojangles) [inline CHANGELOG.md]: relevant?

**Comment by snayler0** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/GeoscienceAustralia/amazonia/pull-request/82/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=GeoscienceAustralia/amazonia)

**Comment by GypsyBojangles** [comment]
Approved

**Pull request merged**